### PR TITLE
Configurable environment variable prefix

### PR DESCRIPTION
**Pull request by joshuacolvin0**
The previous behavior of loading all environment variables with a prefix of `ARBITRUM_` as configuration options runs into a problem if

### CODE REVIEW

File: packages/arb-util/configuration/configuration.go [hunk 1]
```diff
@@ -194,6 +195,8 @@ func ParseValidator(ctx context.Context) (*Config, *Wallet, *ethutils.RPCEthClie
 func ParseNonRelay(ctx context.Context, f *flag.FlagSet) (*Config, *Wallet, *ethutils.RPCEthClient, *big.Int, error) {
 	f.String("bridge-utils-address", "", "bridgeutils contract address")
 
+	f.String("env-prefix", "", "environment variables with given prefix will be loaded as configuration values")
```
Comment (hkalodner) [inline packages/arb-util/configuration/configuration.go:198]: Don't we want to also provide this for relays?

**Review by hkalodner — CHANGES_REQUESTED** [review]

File: packages/arb-util/configuration/configuration.go (hunk 1 above)
Comment (hkalodner) [reply packages/arb-util/configuration/configuration.go:198]: We probably want this in `beginCommonParse`

Comment (joshuacolvin0) [reply packages/arb-util/configuration/configuration.go:198]: :+1:

**Review by hkalodner — APPROVED** [review]
LGTM

**Pull request merged**